### PR TITLE
Prepare e2e-test on prowjob

### PR DESCRIPTION
**Pull request by prksu**
**What this PR does / why we need it**:

This PR adding scripts for running e2e test inside prow

**Special notes for your reviewer**:

/cc @xmudrii @cpanato @MorrisLaw 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: scripts/ci-e2e.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
+mkdir -p "${ARTIFACTS}/logs/"
+
+SSH_KEY_NAME=capdo-e2e-test
+SSH_KEY_PATH=/tmp/${SSH_KEY_NAME}
+create_ssh_key() {
+    echo "generating new ssh key"
+    ssh-keygen -t rsa -f ${SSH_KEY_PATH} -N '' 2>/dev/null <<< y >/dev/null
+    echo "importing ssh key "
+    doctl compute ssh-key import ${SSH_KEY_NAME} --public-key-file ${SSH_KEY_PATH}.pub --access-token ${DIGITALOCEAN_ACCESS_TOKEN}
```
Comment (xmudrii) [inline scripts/ci-e2e.sh:46]: `doctl` recognizes the `DIGITALOCEAN_ACCESS_TOKEN` environment variable, so the flag is not needed.

File: scripts/ci-e2e.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+remove_ssh_key() {
+    local ssh_fingerprint=$1
+    echo "removing ssh key"
+    doctl compute ssh-key delete ${ssh_fingerprint} --access-token ${DIGITALOCEAN_ACCESS_TOKEN} --force
```
Comment (xmudrii) [inline scripts/ci-e2e.sh:52]: `doctl` recognizes the `DIGITALOCEAN_ACCESS_TOKEN` environment variable, so the flag is not needed.

File: test/e2e/suite_test.go [hunk 3]
```diff
@@ -150,9 +150,9 @@ var _ = AfterSuite(func() {
 	// If running in prow, output the logs to the artifacts path
 	artifactPath, exists := os.LookupEnv("ARTIFACTS")
 	if exists {
-		ioutil.WriteFile(path.Join(artifactPath, "capi.log"), []byte(capiLogs), 0644)   // nolint
-		ioutil.WriteFile(path.Join(artifactPath, "cabpk.log"), []byte(cabpkLogs), 0644) // nolint
-		ioutil.WriteFile(path.Join(artifactPath, "capdo.log"), []byte(capdoLogs), 0644) // nolint
+		ioutil.WriteFile(path.Join(artifactPath, "/logs/capi.log"), []byte(capiLogs), 0644)   // nolint
+		ioutil.WriteFile(path.Join(artifactPath, "/logs/cabpk.log"), []byte(cabpkLogs), 0644) // nolint
+		ioutil.WriteFile(path.Join(artifactPath, "/logs/capdo.log"), []byte(capdoLogs), 0644) // nolint
```
Comment (xmudrii) [inline test/e2e/suite_test.go:155]: Are we sure that there are no secrets or something like that in logs?

Comment (prksu) [reply test/e2e/suite_test.go:155]: lemme double check

File: scripts/ci-e2e.sh (hunk 1 above)
Comment (prksu) [reply scripts/ci-e2e.sh:46]: ahh, don't know about this one :grinning: 
will update it

File: test/e2e/suite_test.go (hunk 3 above)
Comment (prksu) [reply test/e2e/suite_test.go:155]: already checked. it only contains state event of component and some info like loadbalancer IP address and machine (name|id)

Comment (xmudrii) [reply test/e2e/suite_test.go:155]: Sounds good in that case.

**Pull request merged**